recipe/conda_build_config.yaml

```
@@ -8,66 +8,53 @@ docker_image:                                # [os.environ.get("BUILD_PLATFORM",
    - quay.io/condaforge/linux-anvil-cuda:10.0       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-   - quay.io/condaforge/linux-anvil-cuda:10.1       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-   - quay.io/condaforge/linux-anvil-cuda:10.2       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
+   - quay.io/condaforge/linux-anvil-cos7-cuda:10.1  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
+   - quay.io/condaforge/linux-anvil-cos7-cuda:10.2  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:11.0       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:11.1       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
    - quay.io/condaforge/linux-anvil-cuda:11.2       # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
 
-   # these two variants are needed for cuTENSOR support
-   - quay.io/condaforge/linux-anvil-cos7-cuda:10.1  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-   - quay.io/condaforge/linux-anvil-cos7-cuda:10.2  # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-
 c_compiler_version:     # [linux]
   - 7                   # [linux64 or aarch64]
   - 8                   # [ppc64le]
 cxx_compiler_version:   # [linux]
   - 7                   # [linux64 or aarch64]
   - 8                   # [ppc64le]
 
-cuda_compiler:   # [linux64]
-  - nvcc         # [linux64]
+cuda_compiler:   # [linux64 or win]
+  - nvcc         # [linux64 or win]
 
 cuda_compiler_version:
-  - None
+  - None                       # [linux64]
   - None                       # [linux64]
   - 9.2                        # [linux64]
   - 10.0                       # [linux64]
   - 10.1                       # [linux64]
-  - 10.2                       # [linux64]
-  - 11.0                       # [linux64]
-  - 11.1                       # [linux64]
-  - 11.2                       # [linux64]
-
-  - 10.1                       # [linux64]
-  - 10.2                       # [linux64]
+  - 10.2                       # [linux64 or win]
+  - 11.0                       # [linux64 or win]
+  - 11.1                       # [linux64 or win]
+  - 11.2                       # [linux64 or win]
 
 cudnn:
-  - undefined
+  - undefined                  # [linux64]
   - undefined                  # [linux64]
```

Should this only be one `undefined`? Also do we want `or win`?

```suggestion
  - undefined                  # [linux64 or win]
```

---

We need 2 `undefined` for Linux, one for base cos6 and the other for base cos7. Since Windows has no such distinction, it only has one `undefined`, hence the `or`.